contents/blog/optimize-to-posthog.md

```diff
@@ -0,0 +1,179 @@
+    posthog.init('<ph_project_api_key>,{api_host:'<ph_instance_address>'})
+</script>
+
+<!-- Optimize -->
+<script src="[https://www.googleoptimize.com/optimize.js?id=OPT_CONTAINER_ID](https://www.googleoptimize.com/optimize.js?id=OPT_CONTAINER_ID)"></script>
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-QCX3G7KSPC"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-QCX3G7KSPC', {'optimize_id': 'GTM-QCX3KSA'});
+</script>
+```
+
+You can also install and initialize PostHog using one of our [13 SDKs](/docs/getting-started/install?tab=sdks) like [React](/docs/libraries/react), [Python](/docs/libraries/python), [Flutter](/docs/libraries/flutter), [Android](/docs/libraries/android), [Node](/docs/libraries/node), and more.
+
+Once you install and initialize PostHog, it automatically starts autocapturing usage of your app and is ready for A/B tests. You don’t need more set up or link with other tools. 
+
+## Creating an A/B test in PostHog
+
+Creating an A/B test in PostHog is different from Optimize. You start by creating your experiment in the [experiments tab](https://app.posthog.com/experiments). Once you click "New experiment," you set up:
+
+- **Name:** Metadata to help you identify your test.
+- **Feature flag key:** How you reference this test in your app or website.
+- **Goal:** The metric trend or conversion you want to improve.
+
+Additional optional setup includes:
+
+- **Description:** Metadata to share information about your test.
+- **Variants:** Enables you to run [A/B/n tests](/tutorials/abn-testing) with more than 2 variants.
+- **Participants:** By default, PostHog rolls experiments out to 100% of users, but you can also use groups.
+- **Secondary metrics:** Non-goal metrics you want to monitor for your test.
+- **Minimum acceptable improvement:** To help you know how long you should run your test.
+
+After completing these, click "Save as draft." You can then implement your experiment in your app.
+
+### How are creating PostHog experiments different than Optimize?
+
+- PostHog doesn’t need an editor page URL. The same experiment can be set up on multiple pages.
```

Comment:
Is "editor page URL" correct? Sounds weird.

Reply:
Yup, its an Optimize term: https://stackoverflow.com/questions/67728889/google-optimize-giving-error-the-editor-page-url-is-matched-by-targeting-rules (Google removed many of their support pages)